addthis.install

```diff
@@ -96,3 +98,12 @@ function addthis_update_7402() {
 
   variable_del('addthis_widget_load_type');
 }
+
+/**
+ * Remove old TWITTER_TEMPLATE_KEY variable, add default value for
+ * TWITTER_TEXT_KEY
+ */
+function addthis_update_7403() {
+  variable_del('addthis_twitter_template');
+  variable_set(AddThis::TWITTER_TEXT_KEY, TWITTER_TEXT_DEFAULT);
```

Comment:
I asked for setting the default but I was wrong. You are already loading a default in the ->getTwitterText(). So thats enough. You can remove this line.